utonics/labproject/main.go

```diff
@@ -94,8 +94,12 @@ func newProject(values map[string]string, botClient, userClient *worker.Client)
 	}
 	msgs = append(msgs, fmt.Sprintf("Team created: %s", team.Name))
 
-	msgs = append(msgs, fmt.Sprintf("Adding user %q to team %q", userClient.UserName, team.Name))
-	botClient.AdminAddTeamMembership(team.ID, userClient.UserName)
+	user, err := userClient.GetSelfInfo()
+	if err != nil {
+		return nil, err
```

Comment:
Why not return `msgs` here as above and below?

Reply:
Yup, should return the messages up to that point.